debezium-connector-postgres/src/main/java/io/debezium/connector/postgresql/UnchangedToastedReplicationMessageColumn.java

```diff
@@ -24,13 +24,23 @@ public class UnchangedToastedReplicationMessageColumn extends AbstractReplicatio
      * Marker value indicating an unchanged TOAST column value.
      */
     public static final Object UNCHANGED_TOAST_VALUE = new Object();
+    public static final Object UNCHANGED_INT_ARRAY_TOAST_VALUE = new Object();
+    public static final Object UNCHANGED_BIGINT_ARRAY_TOAST_VALUE = new Object();
 
-    private boolean isArrayColumn = false;
+    private boolean isStringArrayColumn = false;
+    private boolean isIntArrayColumn = false;
+    private boolean isBigintArrayColumn = false;
 
     public UnchangedToastedReplicationMessageColumn(String columnName, PostgresType type, String typeWithModifiers, boolean optional) {
         super(columnName, type, typeWithModifiers, optional);
         if (typeWithModifiers.equals("text[]") || typeWithModifiers.equals("_text")) {
-            isArrayColumn = true;
+            isStringArrayColumn = true;
```

Comment:
Looking at the code as is know. Doe sit make sense to set a flag as instance attribute and then return a values based on that flag? Maybe the value returned should be set directly to an attribute in constructor and just returned in `getValue`?